index.html

```diff
@@ -120,6 +120,34 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
+        <!-- ________ Lainey's card START ________  -->
+        <div class="card">
+          <p class="name">Lainey</p>
+          <p class="contact">
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/lainey-creighton/" target="_blank">@laineycreighton</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">@laineycreighton</a>
```

Comment:
Hi, you can update this line like this, so the link directs to your github profile, instead of the main page.
You can update this PR by clicking "Commit Suggestion" or make the change in local machine and pushing again.

```suggestion
            <a href="https://github.com/laineycreighton" target="_blank">@laineycreighton</a>
```